src/components/tooltip/tooltip.scss

```diff
@@ -6,66 +6,69 @@ $tooltip-top-margin-lg: rem(1.4) !default;
 $tooltip-top-margin-sm: rem(2.4) !default;
 $tooltip-lr-padding-lg: rem(0.8) !default;
 $tooltip-lr-padding-sm: rem(1.6) !default;
-$tooltip-max-width: rem(3.20) !default;
+$tooltip-max-width: rem(3.2) !default;
```

Comment:
While you are at it, why using root em here? EM is actually for font / typography